### PR TITLE
Add test plan for QUARKUS-710 - Add Support for Micrometer

### PR DESCRIPTION
**Pull request by Sgitario**

### CODE REVIEW

**Comment by Sgitario** [comment]
@rsvoboda let me know your thoughts about this test plan for QUARKUS-710. 
Not sure if covering all the configuration is too much, but maybe we should cover some.

File: QUARKUS-710.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+### Impact on testsuites and testing automation:
+ - OpenShift TS framework needs to be enhanced to install Prometheus
+ - Smoke tests to be added into the OpenShift TS:
+   - Verify expected metrics are shared between the Quarkus application and Prometheus
+ - More test development needs to be added in Beefy-scenarios testsuite:
+   - Scenario with parallel processing of metrics
+   - Scenario with custom tags using `@MeterFilterConstraint` annotation
+   - Scenario to use a custom Prometheus registry
+   - Support for the MicroProfile Metrics API
+   - Scenarios to cover the correct functionality of the following properties:
```
Comment (rsvoboda) [inline QUARKUS-710.md:26]: There are quite a few bullet-points. Which of them are mandatory and which can be treated as optional?

**Review by rsvoboda — CHANGES_REQUESTED** [review]

**Comment by rsvoboda** [comment]
Regarding Prometheus I found these resources:
 - https://docs.openshift.com/container-platform/4.6/monitoring/understanding-the-monitoring-stack.html
 - https://docs.openshift.com/container-platform/4.6/operators/operator_sdk/osdk-monitoring-prometheus.html
 - https://www.openshift.com/blog/configure-openshift-metrics-with-prometheus-backed-by-openshift-container-storage

File: QUARKUS-710.md (hunk 1 above)
Comment (Sgitario) [reply QUARKUS-710.md:26]: All the configuration bullets are optional, so if I understood you correctly, I will remove them.

File: QUARKUS-710.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+### Impact on testsuites and testing automation:
+ - OpenShift TS framework needs to be enhanced to install Prometheus
+ - Smoke tests to be added into the OpenShift TS:
+   - Verify expected metrics are shared between the Quarkus application and Prometheus
+ - More test development needs to be added in Beefy-scenarios testsuite:
+   - Scenario with parallel processing of metrics
+   - Scenario with custom tags using `@MeterFilterConstraint` annotation
+   - Scenario to use a custom Prometheus registry
+   - Support for the MicroProfile Metrics API
```
[the remark this answers is not in the document]
Comment (Sgitario) [reply QUARKUS-710.md:25]: To use the MP Metrics API (annotations).

File: QUARKUS-710.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+## Scope of the testing
+
+Following actions were taken to ensure test coverage and automation for native image:
+ - Define the scope of the testing
+ - Analyze the possible impact on Quarkus itself - e.g performance, compatibility
+ - Determine impact on test suites - e.g. new test suite, increased complexity of existing test suite, maintenance burden
+ - Determine impact on QE infrastructure - e.g. increased complexity of existing automation, maintenance burden, need for new external resources
+
+### Impact on testsuites and testing automation:
+ - OpenShift TS framework needs to be enhanced to install Prometheus
```
[the remark this answers is not in the document]
Comment (Sgitario) [reply QUARKUS-710.md:18]: Updated

File: QUARKUS-710.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+ - Smoke tests to be added into the OpenShift TS:
+   - Verify expected metrics are shared between the Quarkus application and Prometheus
```
[the remark this answers is not in the document]
Comment (Sgitario) [reply QUARKUS-710.md:20]: Added

File: QUARKUS-710.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+ - More test development needs to be added in Beefy-scenarios testsuite:
```
[the remark this answers is not in the document]
Comment (Sgitario) [reply QUARKUS-710.md:26]: Updated

**Pull request merged**